### PR TITLE
Add OpenMDAO 2.6.0 and testflo 1.3.4

### PR DESCRIPTION
**Pull request by bollwyvl**
Adds
- https://github.com/OpenMDAO/OpenMDAO
- https://github.com/OpenMDAO/testflo

## Checklist

- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bollwyvl** [comment]
Not sure what kinds of windows devils this will uncover...

**Comment by bollwyvl** [comment]
Well look at that!

File: recipes/testflo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - testflo = testflo.main:main
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - coverage
+    - python
+    - six
```
Comment (scopatz) [inline recipes/testflo/meta.yaml]: Does this not depend on openmdao?

**Comment by bollwyvl** [comment]
Testflo is the test runner for openmdao... mpi smarts, etc. I couldn't get
all the mpi tests to run, even locally, though! Having the core
functionality in place, even if some of the bells and whistles are not
fully tested, would be really good progress over where this pr started!

**Comment by scopatz** [comment]
AHh I see. This is fine. Thanks!

**Pull request merged**